opencog/cython/PythonSCM.cc

```diff
@@ -97,21 +97,13 @@ void* PythonSCM::init_in_guile(void* self)
 {
 	scm_c_define_module("opencog python", init_in_module, self);
 	scm_c_use_module("opencog python");
-
-	// Make sure that guile and python are using the same atomspace.
-	// This will avoid assorted confusion.
-	AtomSpace* as = SchemeSmob::ss_get_env_as("python-eval");
-	PythonEval& pev = PythonEval::instance(as);
+	PythonEval& pev = PythonEval::instance();
 
 	// This feels hacky, I guess, but I cannot figure out any other
 	// way of telling python which atomspace it is supposed to use
 	// by default.
 	pev.eval("from opencog.atomspace import AtomSpace");
 	pev.eval("from opencog.type_constructors import set_type_ctor_atomspace");
```

Comment:
If you are removing the code below, then please remove the imports, and the comments above, as they are no longer needed. (and teh comments are no longer correct)

Reply:
Ok, i moved it back, seems there is no better solution in absence of c++ functions for the same purpose.